ocean_lib/web3_internal/constants.py

```diff
@@ -14,3 +14,5 @@
 
 MAX_INT256 = 2**255 - 1
 MIN_INT256 = 2**255 * -1
+
+DEFAULT_TOKEN_URI = "data:application/json;base64,eyJuYW1lIjoiT2NlYW4gRGF0YSBORlQiLCJzeW1ib2wiOiJPQ0VBTi1ORlQiLCJkZXNjcmlwdGlvbiI6IlRoaXMgTkZUIHJlcHJlc2VudHMgYW4gYXNzZXQgaW4gdGhlIE9jZWFuIFByb3RvY29sIHY0IGVjb3N5c3RlbS5cblxuVmlldyBvbiBPY2VhbiBNYXJrZXQ6IGh0dHBzOi8vbWFya2V0Lm9jZWFucHJvdG9jb2wuY29tL2Fzc2V0L2RpZDpvcDpmZWVmMDRmOTJmZDliNDhjYTBlNTcxZjYzODM4NDRjMjc1MWRmOWE0YWZkYWY2ZjY3NzlmNmM3OTUwMGZhMmVjIiwiZXh0ZXJuYWxfdXJsIjoiaHR0cHM6Ly9tYXJrZXQub2NlYW5wcm90b2NvbC5jb20vYXNzZXQvZGlkOm9wOmZlZWYwNGY5MmZkOWI0OGNhMGU1NzFmNjM4Mzg0NGMyNzUxZGY5YTRhZmRhZjZmNjc3OWY2Yzc5NTAwZmEyZWMiLCJiYWNrZ3JvdW5kX2NvbG9yIjoiMTQxNDE0IiwiaW1hZ2VfZGF0YSI6ImRhdGE6aW1hZ2Uvc3ZnK3htbCwlM0Nzdmcgdmlld0JveD0nMCAwIDk5IDk5JyBmaWxsPSd1bmRlZmluZWQnIHhtbG5zPSdodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyclM0UlM0NwYXRoIGZpbGw9JyUyM2ZmNDA5Mjc3JyBkPSdNMCw5OUwwLDI5QzksMjQgMTksMTkgMzEsMTlDNDIsMTggNTUsMjMgNjcsMjVDNzgsMjYgODgsMjMgOTksMjFMOTksOTlaJy8lM0UlM0NwYXRoIGZpbGw9JyUyM2ZmNDA5MmJiJyBkPSdNMCw5OUwwLDQzQzksNDUgMTgsNDcgMzAsNDhDNDEsNDggNTQsNDYgNjYsNDVDNzcsNDMgODgsNDMgOTksNDNMOTksOTlaJyUzRSUzQy9wYXRoJTNFJTNDcGF0aCBmaWxsPSclMjNmZjQwOTJmZicgZD0nTTAsOTlMMCw3OEMxMCw3NSAyMCw3MiAzMSw3MUM0MSw2OSA1Myw2OSA2NSw3MEM3Niw3MCA4Nyw3MiA5OSw3NEw5OSw5OVonJTNFJTNDL3BhdGglM0UlM0Mvc3ZnJTNFIn0="
```

Comment:
so all datatokens created by ocean.py will have:
 - name: Ocean Data NFT
 - symbol: OCEAN-NFT
 - external_url: https://market.oceanprotocol.com/asset/did:op:feef04f92fd9b48ca0e571f6383844c2751df9a4afdaf6f6779f6c79500fa2ec
 
 ?
 (cause that is what we are setting:
 ```
 {"name":"Ocean Data NFT","symbol":"OCEAN-NFT","description":"This NFT represents an asset in the Ocean Protocol v4 ecosystem.\n\nView on Ocean Market: https://market.oceanprotocol.com/asset/did:op:feef04f92fd9b48ca0e571f6383844c2751df9a4afdaf6f6779f6c79500fa2ec","external_url":"https://market.oceanprotocol.com/asset/did:op:feef04f92fd9b48ca0e571f6383844c2751df9a4afdaf6f6779f6c79500fa2ec","background_color":"141414","image_data":"data:image/svg+xml,%3Csvg viewBox='0 0 99 99' fill='undefined' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='%23ff409277' d='M0,99L0,29C9,24 19,19 31,19C42,18 55,23 67,25C78,26 88,23 99,21L99,99Z'/%3E%3Cpath fill='%23ff4092bb' d='M0,99L0,43C9,45 18,47 30,48C41,48 54,46 66,45C77,43 88,43 99,43L99,99Z'%3E%3C/path%3E%3Cpath fill='%23ff4092ff' d='M0,99L0,78C10,75 20,72 31,71C41,69 53,69 65,70C76,70 87,72 99,74L99,99Z'%3E%3C/path%3E%3C/svg%3E"}
 ```
 
 
 please generate this on the fly